code/modules/mob/living/carbon/crit.dm

```diff
@@ -0,0 +1,72 @@
+/mob/living/carbon/handle_crit()
+
+	/////////////////////////////////////////////
+	//// cogwerks - critical health rewrite /////
+	/////////////////////////////////////////////
+	//// goal: make crit a medical emergency ////
+	//// instead of game over black screen time /
+	/////////////////////////////////////////////
+	if(getBrainLoss() >= 100 && health < 0)
+		Weaken(30)
+		losebreath += 10
+
+	if(getBrainLoss() >= 120 || (health + (getOxyLoss() / 2)) <= -500)
+		death()
+		return
+
+	if (health <= -100)
+		var/deathchance = min(99, ((getBrainLoss() * -5) + (health + (getOxyLoss() / 2))) * -0.01)
+		if (prob(deathchance))
+			death()
+			return
+
+	if (health < 0 && stat != 2)
+		if(lasthealth > 0)
+			Paralyse(5)
+		if (prob(5))
+			emote(pick("faint", "collapse", "cry","moan","gasp","shudder","shiver"))
+		if (stuttering <= 5)
+			stuttering += 5
+		if (eye_blurry <= 5)
+			blur_eyes(5)
+		if (prob(7))
+			confused += 2
+		if (prob(5))
+			Paralyse(2)
+		switch(health)
+			if (-INFINITY to -100)
```

Comment:
If we are going away from the crit overlays and crit everything can as well go away from static 100 health and make it 
if(-INFINITY to -maxhealth)
if(-maxhealth+1 to -(maxhealth * 0.8))
Use rounding for convinience too maybe
You get the idea right?